### PR TITLE
fix (Db/Kezan) Correct chain of quests.

### PR DESCRIPTION
**Pull request by Asmadeuxx**
### Changes proposed

-  Correct order of start-up missions  goblins
-  Creature Text esES, esMX.

### Issues addressed

- Closes

### Tests performed

-  In-Game Test

### How to Test the Changes:

1. Create a Goblin
2. Start Quests 
3. try to complete

### Known issues and TODO list

 

---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.

### CODE REVIEW

**Comment by pangolp** [comment]
I am writing to you privately, because you need to correct some things.

File: data/sql/updates/pending_db_world/rev_1661826488559382300.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
+
+/*KEZAN QUEST*/
+UPDATE `quest_template_addon` SET `PrevQuestID` = '0' WHERE  `ID` = '14138';
```
Comment (seobryn) [inline data/sql/updates/pending_db_world/rev_1661826488559382300.sql]: @Asmadeuxx  Please remove single quotes in every number

File: data/sql/updates/pending_db_world/rev_1661826488559382300.sql [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+UPDATE `quest_template_addon` SET `PrevQuestID` = '14122' WHERE  `ID` = '14123'; 
+UPDATE `quest_template_addon` SET `PrevQuestID` = '14122' WHERE  `ID` = '14121';
+UPDATE `quest_template_addon` SET `PrevQuestID` = '14122' WHERE  `ID` = '14124'; 
+UPDATE `quest_template_addon` SET `PrevQuestID` = '14124' WHERE  `ID` = '14125';
+UPDATE `quest_template_addon` SET `PrevQuestID` = '14125' WHERE  `ID` = '14126';
+
+INSERT INTO `creature_template_locale` (`entry`, `locale`, `Name`, `FemaleName`, `Title`, `VerifiedBuild`) VALUES
+(34830,'esMX','Trol desafiante','','',18019),
+(34830,'esES','Trol desafiante','','',18019);
+ALTER TABLE `creature_text_locale`
```
Comment (seobryn) [inline data/sql/updates/pending_db_world/rev_1661826488559382300.sql]: What do you want to do with this sentence? O_o.

Please review that.

File: data/sql/updates/pending_db_world/rev_1661826488559382300.sql [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+DROP PRIMARY KEY;
+
+INSERT INTO `creature_text_locale` ( `CreatureID`,`GroupID`,`ID`,`Locale`,`Text`) VALUES
```
Comment (seobryn) [inline data/sql/updates/pending_db_world/rev_1661826488559382300.sql]: I'm not sure if this change has to be located on `broadcast_text_locale`

**Review by seobryn — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1661826488559382300.sql [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+UPDATE `creature_text` SET `BroadcastTextId`=35115 WHERE `CreatureID`=34830 AND `GroupID`=0 AND `ID`=0;
+UPDATE `creature_text` SET `BroadcastTextId`=40315 WHERE `CreatureID`=34830 AND `GroupID`=0 AND `ID`=1;
+UPDATE `creature_text` SET `BroadcastTextId`=35113 WHERE `CreatureID`=34830 AND `GroupID`=0 AND `ID`=2;
+UPDATE `creature_text` SET `BroadcastTextId`=35114 WHERE `CreatureID`=34830 AND `GroupID`=0 AND `ID`=3;
+UPDATE `creature_text` SET `BroadcastTextId`=35112 WHERE `CreatureID`=34830 AND `GroupID`=0 AND `ID`=4;
+UPDATE `creature_text` SET `BroadcastTextId`=40313 WHERE `CreatureID`=34830 AND `GroupID`=0 AND `ID`=5;
+UPDATE `creature_text` SET `BroadcastTextId`=40312 WHERE `CreatureID`=34830 AND `GroupID`=0 AND `ID`=6;
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=19 AND `SourceEntry` IN (14069, 14075,25473,28349,14071,24567,24488,24502,24503,24520,14070,26712,14113,14153,14115,14116,14120,14121,14122,14123,14124,14125,14126);
```
Comment (pangolp) [inline data/sql/updates/pending_db_world/rev_1661826488559382300.sql]: I know it is a minor detail, but while we are at it, leave a space after the comma as in the rest of the records.

**Review by pangolp — APPROVED** [review]
Thanks

**Review by seobryn — APPROVED** [review]

**Pull request merged**